gfx/models/portraits/attachments/male_headgears/secular/goblin/gallywix_hat_01/GallywixHat.asset

```diff
@@ -0,0 +1,19 @@
+pdxmesh = {
+	name = "GallywixHat_mesh"
+	file = "GallywixHat.mesh"
+
+	meshsettings = {
+		name = "GallywixHatShape"
+		index = 0
+		texture_diffuse = "GallywixHatDiffuse.png"
+		texture_normal = "GallywixHatNormal.png"
+		texture_specular = "GallywixHatProperties.png"
+		shader = "standard"
+		shader_file = "gfx/FX/pdxmesh.shader"
+	}
+}
+
+entity = {
+	name = "GallywixHat_entity"
```

Comment:
`male_headgear_secular_goblin_gallywix_hat_01_entity` is an entity I used. Since the name is different, in-game goblins no longer wear hats.